src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -1951,9 +1919,8 @@ function initAugmentations() {
                  "extracellular matrix (ECM). This improves the ECM's ability to " +
                  "structurally support the body and grants heightened strength and " +
                  "durability.<br><br>" +
-                 "This augmentation:<br>" +
-                 "Increases the player's experience gain rate for all combat stats by 5%.<br>" +
-                 "Increases the player's Bladeburner max stamina by 10%.",
+                 "Increases experience gain for all combat stats by 5%.<br>" +
+                 "Increases Bladeburner Stamina by 10%.",
```

Comment:
I'd add the word "max" back in here to be clear: `Bladeburner max stamina`. Same for all of the others